ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/handlers/CqlTransformHandler.java

```diff
@@ -268,7 +274,21 @@ public Object handle(Request request, Response response) throws Exception {
                         !cqlTransformRequest
                             .getHiddenResults()
                             .contains(result.getMetacard().getId()))
-                .collect(Collectors.toList());
+                .collect(toList());
+
+    List<String> metacardIds =
+        results.stream().map(result -> result.getMetacard().getId()).collect(toList());
+
+    Set<String> sourceIds =
+        results
+            .stream()
+            .map(result -> result.getMetacard().getSourceId())
+            .collect(Collectors.toSet());
+
+    securityLogger.audit(
```

Comment:
It looks like this will list the metacard Ids and source Ids but it won't be clear which metacard is from which source right?  Can we update the logging to make this clear?  I think it would be OK if this results in multiple audit log statements.